### PR TITLE
feat: realize json-visualizer with webpack

### PR DESCRIPTION
**Pull request by Ansagan-B**

### CODE REVIEW

File: workflow/json-visualizer-webpack/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    return base;
+}
+
+module.exports = {
+    mode: 'development',
+    entry: {
+        main: './src/index.js',
+        visualizer: './src/visualizer.js'
```
Comment (nastystanevich) [inline workflow/json-visualizer-webpack/webpack.config.js]: why do you create to entries?

File: workflow/json-visualizer-webpack/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    output: {
+        filename: '[name].[contenthash].js',
+        path: path.resolve(__dirname, 'dist'),
+        library: 'webpackNumbers',
```
Comment (nastystanevich) [inline workflow/json-visualizer-webpack/webpack.config.js]: what is the `webpackNumbers` library?

File: workflow/json-visualizer-webpack/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    module: {
+        rules: [
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader']
+            },
+            {
+                test: /\.s[ac]ss$/,
+                use: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader']
+            },
```
Comment (nastystanevich) [inline workflow/json-visualizer-webpack/webpack.config.js:79]: for css use use `style-loader` but for scss `MiniCssExtractPlugin.loader`, why?

**Review by nastystanevich — CHANGES_REQUESTED** [review]
As well add compression of files for production (use `gzip` and `brotliCompress` algoritms)

File: workflow/json-visualizer-webpack/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "json-viewer",
+    "webpack"
+  ],
+  "author": "Ansagan-B <ansagan.gb@gmail.com>",
+  "license": "ISC",
+  "devDependencies": {
+    "clean-webpack-plugin": "^4.0.0",
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "lodash": "^4.17.21",
```
[the remark this answers is not in the document]
Comment (Ansagan-B) [reply workflow/json-visualizer-webpack/package.json]: ok, done!

File: workflow/json-visualizer-webpack/dist/index.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+<div class="additional-info">
+    <h3>Coloring value by type</h3>
+    <ul>
+        <li>Type of <span class="blue">"number": blue</span> color;</li>
+        <li>Type of <span class="red">"string": red</span> color;</li>
+        <li>Type of <span class="purple">"boolean": purple</span> color;</li>
+        <li>Other types: <span class="green">green</span> color.</li>
+    </ul>
+</div>
+</body>
```
[the remark this answers is not in the document]
Comment (Ansagan-B) [reply workflow/json-visualizer-webpack/dist/index.html:33]: I will fix it

**Review by nastystanevich — APPROVED** [review]

File: workflow/json-visualizer-webpack/src/visualizer.js [hunk 6]
```diff
@@ -1,12 +1,14 @@
...
+import _isArray from 'lodash/isArray';
 
 const result = document.getElementById('result');
 
 export default class JsonVisualizer {
 
     parseJSONString(str) {
         try {
-            if (_.isNull(str)) {
+            if (str === 'null') {
```
[the remark this answers is not in the document]
Comment (Ansagan-B) [reply workflow/json-visualizer-webpack/src/visualizer.js:11]: _isNull has some issues with verification at this point

**Pull request merged**